crates/ruma-events/src/dummy.rs

```diff
@@ -14,7 +14,7 @@ use crate::BasicEvent;
 /// this *m.dummy* event as the most recent event and using the keyshare request to set up the
 /// session. The keyshare request and *m.dummy* combination should result in the original
 /// sending client receiving keys over the newly established session.
-pub type DummyEvent = BasicEvent<DummyEventContent>;
+pub type DummyEvent = BasicToDeviceEvent<DummyEventContent>;
```

Comment:
I wonder whether we should just delete this type alias. I don't think the notion of "BasicToDeviceEvent" makes sense, I would expect this type alias to be used in none of the enums, right?

Reply:
That's true. I was wondering if I would break anything if I removed it but it looks like it won't.